os/pm/pm_metrics.c

```diff
@@ -82,6 +93,15 @@ static void pm_print_metrics(double total_time, int n_domains)
 	}
 	pmdbg("\n");
 	pmdbg("\n");
+	pmdbg(" PROCESS NAME | WAKEUP COUNTS | MIN SLEEP TIME \n");
+	pmdbg("--------------|---------------|----------------\n");
+	for (index = 0; index < CONFIG_MAX_TASKS; index++) {
+		if (g_pm_metrics->thread_metrics.wakeup_counts[index]) {
+			pmdbg(" %12s | %13d | %12dms \n", sched_gettcb(index)->name, g_pm_metrics->thread_metrics.wakeup_counts[index], TICK2MSEC(g_pm_metrics->thread_metrics.min_sleep_ticks[index]));
+		}
+	}
+	pmdbg("\n");
+	pmdbg("\n");
 	pmdbg(" BOARD STATE | PM STATE |          TIME          \n");
 	pmdbg("-------------|----------|------------------------\n");
 	pmdbg(" %11s | %8s | %10dms (%6.2f%%) \n", "WAKEUP", "NORMAL", TICK2MSEC(g_pm_metrics->state_metrics.state_accum_ticks[0]),\
```

Comment:
Suggest to change "WAKEUP" -> "ACTIVE"

Reply:
```
pmdbg(" %11s | %8s | %10dms (%6.2f%%) \n", "SLEEP", "SLEEP", TICK2MSEC(g_pm_metrics->board_sleep_ticks),\
	((double)g_pm_metrics->board_sleep_ticks) * 100.0 / total_time);
```
Suggest to change "SLEEP" -> "INACTIVE"

Reply:
So, system active corresponds one of 3 PM states.
System inactive corresponds to PM_SLEEP